samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AuthorImportExport_eu.properties

```diff
@@ -61,3 +61,7 @@ file_is=Fitxategi hau honela dago\:
 from_samigo=Esportatu sistema honetatik (edo beste Sakai sistema batetik)
 from_respondus=Respondus-etik esportatzeko modua  (\u0080\u009c Respondus-en "Gorde QTI XML fitxategia" aukeratzen duzunean, hautatu "Puntuak zenbaki dezimaletarako" galdera bakoitzari eman zaion balioa gordetzeko)  \u0080.
 
+## S2U-25 Question pools export
+export_pool=Esportatu biltzaileak
+importExport_pool_warning2=Las bater\u00edas que contengan <b>\u201cOpci\u00f3n m\u00faltiple\u201d - \u201cVarias correctas, seleccionar varias\u201d</b> con la opci\u00f3n de <b>\u201cTodas o ninguna\u201d</b>, no conservar\u00e1n esta configuraci\u00f3n tras la exportaci\u00f3n a otro sitio. Tras la importaci\u00f3n tendr\u00e1 que editarlo de nuevo antes de su publicaci\u00f3n si quiere conservar esta opci\u00f3n de <b>\u201cTodas o ninguna\u201d</b>.
```

Comment:
@PabloSanRoman would you mind contributing a translation? Thanks!

Reply:
Hello @jesusmmp , @mpellicer 

export_pool=Esportatu biltzailea
importExport_pool_warning2= Biltzaileak baldin badu galdera mota hau, “Zenbait aukera > Zenbait ongi, batzuk hautatzeko > “Dena edo ezer ez” aukerarekin, esportatu ondoren aukera hau aldatuko da. Inportatu ondoren eskuz editatu beharko duzu “Dena edo ezer ez” aukera mantentzeko. 
cp_pool_message= IMS Content Packaging aukeratu beharko duzu baldin eta biltzaileak erantsitakoren bat badu, edo HTML estekaren bat badago kanpoko fitxategi bat lotzeko. 

Saludos!!

Reply:
Just added! Thanks!